### PR TITLE
Format python now that auto-formatting is actually enabled

### PR DESCRIPTION
**Pull request by Lohrer**
# What does this change intend to accomplish?

See title. 

### Checklist

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Does your submission pass linting & tests? You can test on localhost using `./scripts/test`

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -41,10 +41,11 @@ good-names=i,
 # Number of spaces of indent required inside a hanging or continued line.
 indent-after-paren=2
 
-# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
-# tab).
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1 tab).
 indent-string='  '
 
+max-line-length=120
```
Comment (linknum23) [inline .pylintrc:47]: :heart:

Comment (Lohrer) [reply .pylintrc:47]: You're welcome haha. Note that autopep8 was set to a line length of 120 recently, but pylint's setting got missed.

**Review by linknum23 — COMMENTED** [review]
We may want to check with @rtertiaer to verify that auto-formatting works for his vi-based development workflow.

**Comment by linknum23** [comment]
Note: I had to update my python version to get this to work. Using  python 3.7, the same version on amplipi has failures with the jedi server.

**Review by linknum23 — APPROVED** [review]

**Pull request merged**